.github/pull_request_template.md

```diff
@@ -1,19 +1,19 @@
 ### ☑️ Resolves
 
-- Fix #…
+-   Fix #…
 
 ### 🖼️ Screenshots
 
-🏚️ Before | 🏡 After
----|---
-B | A
+| 🏚️ Before | 🏡 After |
+| --------- | -------- |
+| B         | A        |
 
 ### 🚧 Tasks
 
-- [ ] ...
+-   [ ] ...
 
 ### 🏁 Checklist
 
-- [ ] ⛑️ Tests are included or are not applicable
-- [ ] 📘 Component documentation has been extended, updated or is not applicable
-- [ ] 3️⃣ Backport to `next` requested with a Vue 3 upgrade
+-   [ ] ⛑️ Tests are included or are not applicable
+-   [ ] 📘 Component documentation has been extended, updated or is not applicable
+-   [ ] 3️⃣ Backport to `next` requested with a Vue 3 upgrade
```

Comment:
Seems weird. Does it put three spaces to be in line with a header?

Reply:
No it aligns it with the tab width for this:
```md
-   First point
    Also the same point
-   Second point
```